pkg/modules/filters.go

```diff
@@ -70,7 +74,7 @@ func FilterRegex(cal *ics.Calendar, params map[string]string) ([]int, error) {
 				}
 			}
 
-			if regex.MatchString(target) {
+			if ret, _ := regex.MatchString(target); ret == true {
```

Comment:
```suggestion
			if ret, _ := regex.MatchString(target); ret {
```

we don't need the explicit check since `ret` is always a boolean (https://pkg.go.dev/github.com/dlclark/regexp2#Regexp.MatchString)